storage/simulation/cluster.go

```diff
@@ -272,21 +296,24 @@ func (c *Cluster) String() string {
 	return buf.String()
 }
 
-// StringEpochHeader creates the string header for epoch outputs based on all
-// of the current stores.
-func (c *Cluster) StringEpochHeader() string {
-	var buf bytes.Buffer
-	buf.WriteString("Store:\t")
+// OutputEpochHeader outputs to the epoch writer the header for epoch outputs
+// based on all of the current stores.
+func (c *Cluster) OutputEpochHeader() {
+	w := new(tabwriter.Writer)
```

Comment:
why are you making a new tabwriter each time? do you ever write to `epochWriter` using a non-tabwriter?